docs/source/online_portfolio_selection/index.rst

```diff
@@ -0,0 +1,73 @@
+.. _online_portfolio_selection-index:
+
+==========================
+Online Portfolio Selection
+==========================
+
+.. toctree::
+    :maxdepth: 4
+    :caption: Benchmarks
+    :hidden:
+
+    benchmarks/index
+
+.. toctree::
+    :maxdepth: 4
+    :caption: Momentum
+    :hidden:
+
+    momentum/index
+
+.. toctree::
+    :maxdepth: 4
+    :caption: Mean Reversion
+    :hidden:
+
+    mean_reversion/index
+
+.. toctree::
+    :maxdepth: 4
+    :caption: Pattern Matching
+    :hidden:
+
+    pattern_matching/index
+
+There are four different strategies currently implemented in the Online Portfolio Selection module.
+
+1. Benchmarks
+
+2. Momentum
```

Comment:
Perhaps worth moving this part to the beginning of the **Online Portfolio Selection** docs module so users have an idea of the parent class used in the **Benchmarks** section.

I assume you planned to make it this way, but currently, the order of elements in the complied HTML is:
- ONLINE PORTFOLIO SELECTION
    - Benchmarks
    - Momentum
    - Mean Reversion
    - Pattern Matching
    - Online Portfolio Selection
    - Universal Portfolio